iib/workers/tasks/build_merge_index_image.py

```diff
@@ -65,12 +65,17 @@ def _add_bundles_missing_in_source(
     :param str overwrite_target_index_token: the token used for overwriting the input
         ``source_from_index`` image. This is required to use ``overwrite_target_index``.
         The format of the token must be in the format "user:password".
-    :return: bundles which were added to the index image.
-    :rtype: list
+    :return: tuple of bundles which were added to the index image and the bundles in the new index
```

Comment:
Suggestion: `tuple where the first value is a list of bundles.... and the second value is a list of bundles...`